src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -13,6 +13,10 @@ const listItemStyles: IComponentPartStylesInput<IListItemProps, any> = {
         cursor: 'pointer',
       },
     }),
+    ...(selected && {
+      background: 'rgba(98, 100, 167, .8)',
```

Comment:
Not sure selected should be prop or managed state. 
But definitely not hard coded values...

Reply:
Okay! Are you suggesting I should use colors from siteVariables?

Reply:
Brand06 seems similar - perhaps that - design can review